learning_assistant/models.py

```diff
@@ -35,7 +35,15 @@ class LearningAssistantMessage(TimeStampedModel):
     .. pii_retirement: third_party
     """
 
+    USER_ROLE = 'user'
+    ASSISTANT_ROLE = 'assistant'
+
+    Roles = (
+        (USER_ROLE, USER_ROLE),
+        (ASSISTANT_ROLE, ASSISTANT_ROLE),
+    )
+
     course_id = CourseKeyField(max_length=255, db_index=True)
     user = models.ForeignKey(USER_MODEL, db_index=True, on_delete=models.CASCADE)
-    role = models.CharField(max_length=64)
+    role = models.CharField(choices=Roles, max_length=64)
```

Comment:
I think this doesn't affect the database, so no migration needed.

Reply:
Are you sure? Did you run `makemigrations`?

Reply:
I didn't. I'm sorry for that.
Just added the migration file.